reportportal/v5/templates/api-deployment.yaml

```diff
@@ -73,6 +73,8 @@ spec:
           value: "minio"
         - name: RP_BINARYSTORE_MINIO_ENDPOINT
           value: "{{ .Values.minio.endpoint }}"
+        - name: RP_BINARYSTORE_MINIO_REGION
```

Comment:
would it be better to wrap it with some 'IF' statement?

Reply:
Done

Reply:
Not done:
```       
{{ if .Values.minio.region }}
```

Reply:
Aws value deleted